src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend_utils.py

```diff
@@ -63,14 +56,7 @@ def _get_model_caching(options) -> Optional[Any]:
         caching = options["model_caching"]
         if bool(caching) and str(caching).lower() not in ["false", "0"]:
             return True
-        else:
-            return False
-    else:
-        caching = os.getenv("OPENVINO_TORCH_MODEL_CACHING")
```

Comment:
please update docs if you remove env vars https://github.com/openvinotoolkit/openvino/blob/a1de1908bfd5f597d48422f21796daa3bfd08120/docs/articles_en/openvino-workflow/torch-compile.rst#L117

Reply:
Yes maxim, I have told Karol about this already we are going to raise a PR with the docs change